index.html

```diff
@@ -40,3 +40,166 @@ <h2 id="connect">LET'S CONNECT</h2>
 </ul>
                 
     </section>
+    <section id ="my-works">
+    <div class="all-content">
+        
+            <img src="media/Snapshoot Portfolio.png">
+        
+        <h1 id ="project-title">
+            Tonic
+        </h1>
+               
+            <ul class="project-summary">
+                <li id="canopy">CANOPY</li>
+                <li id="back-end"><img src="/media/Counter.png" alt="counter">Back End Dev</li>
+                <li id="back-end"><img src="/media/Counter.png" alt="counter">2023</li>
```

Comment:
- [ ] These images are not appearing on your page. Please fix this by adding a `.` at the beginning of the file path  as shown `<li id="back-end"><img src="./media/Counter.png" alt="counter">Back End Dev</li>` > do this for all images that are not appearing on the page
_**figma**_
![blanks-1](https://user-images.githubusercontent.com/99656687/213684565-15099220-f6e7-4108-b302-05187e6f3720.png)

_**your design**_
![blanks-2](https://user-images.githubusercontent.com/99656687/213684596-cb051e5d-33f7-46b6-bfdf-452ce32f6591.png)